evidence/pool_test.go

```diff
@@ -14,6 +15,13 @@ import (
 
 var mockState = sm.State{}
 
+func TestMain(m *testing.M) {
+	types.RegisterMockEvidences(cdc)
+
+	code := m.Run()
```

Comment:
Why the extra assignment instead of passing `m.Run` straight to `os.Exit`?

Reply:
in my case it's easier to follow the execution path. in the case of `os.Exit(m.Run())`, the accent is shifted towards os.Exit, which should not be the case